### PR TITLE
Update Popper.js to 1.11.0

### PR DESCRIPTION
**Pull request by Johann-S**
Update Popper.js to have those awesome bugfixes : https://github.com/FezVrasta/popper.js/releases/tag/v1.11.0

IMO this PR should be merged before our first beta @mdo
and I removed the line which add an error because we don't have Popper's `.js.map` file @XhmikosR 😉

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3267,9 +3267,9 @@
       "dev": true
     },
     "popper.js": {
-      "version": "1.10.8",
```
Comment (XhmikosR) [inline package-lock.json]: Are you sure there are no other deps due to the popper.js update?

Comment (Johann-S) [inline package-lock.json]: Popper.js has no deps so it shouldn't have other deps

Comment (bardiharborow) [inline package-lock.json]: Popper.js has no dependencies.

Comment (XhmikosR) [inline package-lock.json]: Right. Looks good to me then. 👍

**Review by XhmikosR — APPROVED** [review]

**Review by bardiharborow — APPROVED** [review]
Approved on the code. I can't offer an opinion as to whether this should ship in Beta 1 or 2.

**Pull request merged**